GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -45,6 +45,9 @@ public partial class RevisionFileTreeControl : GitModuleControl
 
 See the changes in the commit form.");
 
+        private readonly TranslationString _cantSaveToTemp = new("Unable to save to Temp directory.");
+        private readonly TranslationString _cantExtractTempArchive = new("Unable to extract tempory archive.");
```

Comment:
I think you mean `temporary`?
Also commit the translation file if you want to fix the build.
See https://github.com/gitextensions/gitextensions/wiki/Translations#net-50-v36

Reply:
Thanks for your comments, have fixed the typo and updated the translation file.